src/test/scala/uk/gov/hmrc/test/ui/cucumber/utils/MongoConnection.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.test.ui.cucumber.utils
+
+import org.mongodb.scala.MongoClient
+import org.mongodb.scala.bson.collection.immutable.Document
+import play.api.libs.iteratee.Done
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+object MongoConnection {
+
+  private val timeout: FiniteDuration = 10.seconds
+
+  private val mongoClient: MongoClient = MongoClient()
+
+  sys.addShutdownHook {
+    mongoClient.close()
+  }
+
+  def dropMongoCollection(db: String, collection: String): Unit =
+    try Await.result(
+      mongoClient
+        .getDatabase(db)
+        .getCollection(collection)
+        .drop()
+        .head(),
+      timeout
+    )
+    catch {
+      case e: Exception => println("Error: " + e)
+    }
+
+  def insert(source: List[String], database: String, collection: String): Unit = {
+
+    dropMongoCollection(database, collection)
```

Comment:
Thinking about this one, do we want to drop the full collection during tests, or just the registrations that the tests add. Im just thinking if people have registrations locally that they use for testing we will end up having to do them over and over, let me know what you think

Reply:
We do the drop on the registration journey tests so that we don't get duplicates. This kind of means I'm used to not being able to keep registrations in my db for too long so I guess it depends on how you guys feel? We don't HAVE to have it in here but it might limit the chance of getting problems with duplicates?

Reply:
Maybe we can implement some VRNs pattern that keeps them separate. I personally would rather not have to run through a registration every time I need to check something on returns (time is prescious). Maybe all ATs start with 777 (random example) and then we know not to use them locally. But the drop is still good, just maybe drop ones related to ATs

Reply:
Ok no problem. It now just deletes the one registration we are inserting for these tests :)